en_US/modules/slow_subscribers_statistics.md

```diff
@@ -1,86 +1,94 @@
+
 # Table of Contents
 
-1.  [Slow subscribers statistics](#org10c5186)
-    1.  [Create module](#org395e425)
-    2.  [Implementation note](#orgdd4d5d6)
-        1.  [Latency calculation method](#org693a7e9)
-        2.  [Average latency calculation method](#org712747a)
-    3.  [Configuration description](#orgcd5b688)
-    4.  [Slow subscribers record](#orgb3ce581)
+1.  [Slow subscribers statistics](#org0a58d32)
+    1.  [open module](#org7939dfc)
+    2.  [Implementation note](#org417d240)
+    3.  [Configuration description](#orgf0feb6e)
+    4.  [Slow subscribers record](#orga6267c1)
+
 
+<a id="org0a58d32"></a>
 
-<a id="org10c5186"></a>
+# Slow subscribers statistics
 
-## Slow subscribers statistics
+This module ranks subscribers and topics in descending order according to the latency of message transmission
 
-This function ranks subscribers in descending order according to the average latency of message transmission
 
-<a id="org395e425"></a>
+<a id="org7939dfc"></a>
 
-### Create module
+## Open module
 
-Open EMQ X Dashboard, click on the "Module" on the left. Then, select "add module":
+Open EMQ X Dashboard, click on the "Module" on the left. 
 
 ![image](./assets/slow_subscribers_statistics_1.png)
 
-Select the **Slow Subscribers Statistics** module, and then click *Start*
+Then, select the **Slow Subscribers Statistics** module, and then click *Enable*
 
-<a id="orgdd4d5d6"></a>
 
-### Implementation note
+<a id="org417d240"></a>
 
-This function will track the time consumption of QoS1 and QoS2 messages to complete the whole process of message transmission after arriving at EMQX, and then use the exponential moving average algorithm to calculate the average message transmission latency of the subscriber, and then rank the subscribers according to the latency.
+## Implementation note
 
-Since QoS1 and QoS2 messages may fail to complete the transmission process due to various reasons at the same time, this function will also try to add subscribers to the ranking according to the time of expiration.
+This function will track the time consumption of the entire message transmission process after the QoS1 and QoS2 messages arrive at EMQX, and then calculate the message transmission latency according to the options in the configuration.
```

Comment:
With this feature enabled, EMQ X will start measuring message transmission latency. The measurement always starts at when a message is received by EMQ X, the end of the measurement is configurable. See more details….